workflow-templates/docs.yml

```diff
@@ -0,0 +1,21 @@
+# Builds the Sphinx documentation in the docs folder
+
+---
+name: docs
```

Comment:
Possibly "Sphinx" / "sphinx.yml" (not sure how we're doing on capitalization standardization)

Reply:
I was going with the name of the folder i.e. ``/docs`` but happy to switch if it is clearer

Reply:
I've been going with lowercase for the filename and uppercase for the name but I am fine with `name: sphinx`

Reply:
files renamed